docs/Gemfile

```diff
@@ -7,17 +7,17 @@ source "https://rubygems.org"
 #
 # This will help ensure the proper Jekyll version is running.
 # Happy Jekylling!
-gem "jekyll", "~> 4.3.2"
+gem "jekyll", "~> 4.3.4"
 
-# This is the default theme for new Jekyll sites. You may change this to anything you like.
-gem "minima", "~> 2.5"
+# Theme
+gem "owasp-td-jekyll", "~> 0.2.1"
```

Comment:
This was last updated 4 years ago (man, my grays are starting to show).  I had archived it where we weren't using it.  Would it make sense for me to do a gemfile update and publish a new version?

https://github.com/lreading/owasp-td-jekyll

Reply:
Looks like you and I were on the same page @jgadsden !

I've unarchived the repo and will see what I can do about updating the gemfile.  I can only assume it needs an update.  Given that the ouput is a static website, I think it's probably fine to continue without the updates?  I'll leave that up to you!

Reply:
I pushed an update: https://rubygems.org/gems/owasp-td-jekyll 
The newest version is 0.2.2 if we'd like to use that one.  The only difference is updated dependencies.  I ran the theme locally and all looked good to me. :sunglasses:

Reply:
The gemfile.lock was deleted, so I updated the .gitignore to include it and pushed it with the change to the Gemfile